### PR TITLE
Fix data loss

### PR DESCRIPTION
**Pull request by Paul-Licameli**
Resolves: #3447

*(short description of the changes and the motivation to make the changes)*

- [x] I signed [CLA](https://www.audacityteam.org/cla/)
- [x] The title of the pull request describes an issue it addresses
- [x] If changes are extensive, then there is a sequence of easily reviewable commits
- [x] Each commit's message describes its purpose and effects
- [x] There are no behavior changes unnecessary for the stated purpose of the PR

Recommended:
- [x] Each commit compiles and runs on my machine without known undesirable changes of behavior

### CODE REVIEW

File: src/effects/RealtimeEffectStateUI.cpp [hunk 1]
```diff
@@ -156,3 +166,7 @@ void RealtimeEffectStateUI::UpdateTitle()
    mEffectUIHost->SetName(title);
 }
 
+void RealtimeEffectStateUI::AutoSave(AudacityProject &project)
+{
+   ProjectHistory::AutoSave::Call(project);
```
Comment (crsib) [inline src/effects/RealtimeEffectStateUI.cpp:171]: I still observe this call during the playback. I had an impression that this is wrong?

Comment (Paul-Licameli) [reply src/effects/RealtimeEffectStateUI.cpp:171]: Playback?  Which steps to reproduce are you referring to?

Comment (crsib) [reply src/effects/RealtimeEffectStateUI.cpp:171]: Hmm, looks like I've missed something. So this change is really about moving ProjectWindowDestroyedMessage?

Comment (Paul-Licameli) [reply src/effects/RealtimeEffectStateUI.cpp:171]: Updated steps to reproduce at https://github.com/audacity/audacity/issues/3663 explain that playback was not a necessary part

**Review by crsib — APPROVED** [review]

**Pull request merged**